src/pkg/cli/client/byoc/do/stream.go

```diff
@@ -90,6 +91,14 @@ func (bs *byocServerStream) Receive() bool {
 }
 
 func (bs *byocServerStream) Err() error {
+	var closeErr *websocket.CloseError
```

Comment:
See if `errors.Is(err, io.ErrUnexpectedEOF)` works

Reply:
Unfortunately, it does not seem like the error wraps `io.EOF` or `io.UnexpectedEOF` : (